.gitignore

```diff
@@ -13,3 +13,12 @@ target/
 **/*.classpath
 **/*.project
 **/*.settings
+
+# Ignore Gradle project-specific cache directory
+.gradle/
+build/generator/out/
```

Comment:
Does this need to be here?
If so why is there not a mention of `build/schemas/out`

Reply:
Remnant of changing gradle build settings. Removed.